scripts/screeps_rl_simulator_harness.py

```
@@ -3359,6 +3614,206 @@ def run_self_test(stdout: TextIO = sys.stdout) -> int:
+        bot_commit=bot_commit,
+        wall_clock_seconds=0.0,
+    )
+    artifact["variants"] = variant_results
+    artifact["resourceGuard"] = resource_guard
+    artifact["cleanup"] = cleanup
+    artifact["failurePhase"] = phase
+    artifact["failureArtifactPath"] = str(failure_path)
+    artifact["ownedRoomScorecard"] = build_run_owned_room_scorecard(run_id, variant_results)
+    artifact["ownedRoomScorecardPath"] = str(scorecard_path)
+    validate_run_artifact(artifact)
+    secret_values = dataset_export.configured_secret_values() + [os.environ.get("STEAM_KEY", "")]
```

Ruff (0.15.12)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)